packages/react-graphql/package.json

```diff
@@ -27,7 +27,9 @@
     "@shopify/async": "^2.0.0",
     "@shopify/react-async": "^3.0.2",
     "@shopify/react-hooks": "^1.2.1",
+    "@shopify/react-network": "^3.1.3",
     "@shopify/useful-types": "^1.3.0",
+    "apollo-cache-inmemory": "^1.6.2",
```

Comment:
this package is only for type, we can do dev dependency

Reply:
The `InMemoryCache` class is also used
https://github.com/Shopify/quilt/blob/c3b3a09b0953d2090a56fd4e597cbc9c54932248/packages/react-graphql/src/createClient.ts#L23

Reply:
nvm!